cluster_experiments/power_config.py

```diff
@@ -122,6 +126,88 @@ class PowerConfig:
 
     seed: Optional[int] = None
 
+    def __post_init__(self):
```

Comment:
Wdyt about adding this method:

```python
    def set_and_log(self, attr, value, other_attr):
        setattr(self, attr, value)
        logging.warning(
            f"{attr} = {getattr(self, attr)} has no effect with "
            f"{other_attr} = {getattr(self, other_attr)}. "
            f"Overriding {attr} to {value}."
        )
```

then doing:

```python
        if "switchback" not in self.splitter:
            if self._are_different(self.switch_frequency, None):
                self.set_and_log("switch_frequency", None, "splitter")
```

Reply:
Looks nice! I'll give it a try